kernelci/api/latest.py

```diff
@@ -167,6 +167,15 @@ def get_user_profiles(
         return self._get_api_objs(params=params, path='users/profile',
                                   limit=limit, offset=offset)
 
+    def create_user(self, username: str, password: str, profile: dict) -> dict:
+        data = {
+            'password': password,
+        }
+        params = {
+            'email': profile['email'],
+        }
```

Comment:
This should also have a provision to add the user to a group.

Reply:
Yes, however I think we're missing the part about creating groups and listing groups etc.  So maybe this can be ignored for now as we need to have this feature merged today to be able to create user accounts for the early access phase next week.

Reply:
OK I've added an issue to keep track of that: #2084

Reply:
We can use `kci user find_groups` to list available groups.
But yes we need to add a command to create a user group before we can add users to new groups.
Thanks for creating the issue.